java/ql/src/Security/CWE/CWE-917/OgnlInjection.ql

```diff
@@ -11,10 +11,8 @@
  */
 
 import java
-import semmle.code.java.dataflow.FlowSources
-import DataFlow
+import semmle.code.java.security.OgnlInjectionQuery
 import DataFlow::PathGraph
-import OgnlInjectionLib
 
 from DataFlow::PathNode source, DataFlow::PathNode sink, OgnlInjectionFlowConfig conf
 where conf.hasFlowPath(source, sink)
```

Comment:
The alert message (in the line below this, but I can't comment on that line as it is just outside the diff) could be improved (the docs team may have suggestions here), as-is it reads a bit awkward with the double use of the word "input".

Reply:
I'd recommend changing to:

```
select sink.getNode(), source, sink, "OGNL expression might include data from $@.",
  source.getNode(), "this user input"
```